src/components/DetailsList/DetailsList.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IDetailsListProps extends React.Props<DetailsList> {
   /** The items to render. */
   items: any[];
 
+  /**
+   * Optional default focused index to set focus to once the items have rendered and the index exists.
+   */
+  initialFocusedIndex?: number;
```

Comment:
Definitely not. Undefined will avoid focusing on anything, which should be the default behavior. This is an opt-in feature that should only be used in specific scenarios where the list is intended to be the primary one on the page.